bin/rrd2whisper.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/home/rrawdon/git/whisper-env/bin/python3
```

Comment:
This shouldn't be changed

Reply:
oh geez, that's what I get for working off of an installed copy rather than my git copy.  Will update shortly.

Reply:
Check out setuptools develop mode or with `pip install -e .`. You won't need for copies any more.